pom.xml

```diff
@@ -33,6 +33,7 @@
         <module>sample-service</module>
         <module>practitioner-service</module>
         <module>config-server</module>
+        <module>discovery-server</module>
```

Comment:
can you move both of these below the "core" services?

Reply:
Done